src/main/java/com/minecolonies/sounds/CitizenSounds.java

```diff
@@ -0,0 +1,54 @@
+package com.minecolonies.sounds;
+
+import com.minecolonies.util.SoundUtils;
+import net.minecraft.util.SoundEvent;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.World;
+
+import java.util.Random;
+
+/**
+ * Class used to store the basic citizen sounds.
+ */
+public class CitizenSounds
+{
+    /**
+     * Chance to say a phrase.
+     */
+    private static final int PHRASE_CHANCE = 50;
+
+    /**
+     * The citizen sound events for the females.
+     */
+    public static class Female
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a private constructor to hide the implicit public one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AS1118)